Assets/UI/WorldView/citybannermanager_CQUI.lua

```diff
@@ -22,11 +22,6 @@ BASE_CQUI_CityBanner_UpdateRangeStrike = CityBanner.UpdateRangeStrike;
 local COLOR_CITY_GREEN           = UI.GetColorValueFromHexLiteral(0xFF4CE710);
 local COLOR_CITY_RED             = UI.GetColorValueFromHexLiteral(0xFF0101F5);
 local COLOR_CITY_YELLOW          = UI.GetColorValueFromHexLiteral(0xFF2DFFF8);
-local BANNERTYPE_CITY_CENTER     = 0;
```

Comment:
these are declared as global values in the unmodified firaxis code and can be consumed by us (in fact, had been already for the mountain tunnel and the qh..one)